index.html

```diff
@@ -21,45 +21,70 @@
+    <link rel="stylesheet" href="desktop.css">
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <section class="wrap-section">
+    <section class="wrapper">
         <header>
             <nav>
-                <a href="#" class="my-logo">Aneal Laryea</a>
-                <a href="#"><img src="assets/image/menn1.png" alt="menu"></a>
+                 <a href="#" class="my-logo">Aneal Laryea</a>
+                <!--<a href="#"><img src="assets/image/menn1.png" alt="menu"></a> -->
+                <!-- <div class="logo">
+                    <a href="#" class="my-logo">Aneal Laryea</a>
+                  </div> -->
+                
+                <ul class="sects">
+                    <li class="port"><a href="#">Portfolio</a></li>
+                    <li class="aabout mp dbx"><a href="#">About</a></li>
+                    <li class="contact mp dbx"><a href="#">Contact</a></li>
+                    <li class="union mp dbx"><a href="#"><img src="assets/image/Union.png" alt="message"></a></li>
+                    <li><a href="#"><img src="assets/image/menn1.png" alt="menu" class="dp"></a></li>
+                </ul>
+                
             </nav> 
-        </header>
-    
-        <section class="headline">
+        
+         <div class="heady">
+        <div class="headline">
             <div class="primary-text">
-                <h1>Hey There.
+                <h1 class="hred">Hey There.
                     I’m Aneal</h1>
             </div>
             <div class="supporting-text">
                 <h2 class="sofew">I’am a Software Developer</h2>
             </div>
             <div class="description">
-                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                <p class="dms">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                     don’t hestiate to contact me.</p>
             </div>
-            <div class="social-media">
-                <ul class="socials">
+            <div class="mmed">
+                <ul class="ussm">
                     <li class="ss"><img src="assets/image/github.png" alt="github icon"></li>
                     <li class="ss"><img src="assets/image/linkdin.png" alt="linkdin icon"></li>
-                    <li class="ss"><i class="fa-brands fa-angellist"></i></li>
-                    <li class="ss"><i class="fa-brands fa-twitter"></i></li>
-                    <li class="ss"><i class="fa-solid fa-m"></i></li>
+                    <li class="ss"><img src="assets/image/angel1.png" alt="angelist icon"></li>
+                    <li class="ss"><img src="assets/image/twit1.png" alt="twitter icon"></li>
+                    <li class="ss"><img src="assets/image/med1.png" alt="medium icon"></li>
```

Comment:
These social icons should be linked to social profiles. Kindly use `a` tags to make them links.
![head-links](https://user-images.githubusercontent.com/35504149/172962021-2a951474-221e-4d25-9086-c4a662361b61.png)

Reply:
* a tags linked to social profiles.